README.md

```diff
@@ -11,44 +11,49 @@ See also [How to install Dryad](http://wiki.datadryad.org/How_To_Install_Dryad#B
 
 These applications must be installed on your host machine.  They will be used to build and run a virtual machine containing Dryad.
 
-1. [Vagrant](http://vagrantup.com)
-2. [Ansible](http://ansible.com)
-3. [VirtualBox](http://virtualbox.org)
+1. [Vagrant](http://vagrantup.com) (version 2.0 or higher)
+2. [Ansible](http://ansible.com) (version 2.0 or higher)
+3. [VirtualBox](http://virtualbox.org) (not required if you will be hosting the VM on the Amazon AWS cloud)
 
 Vagrant and VirtualBox installation packages can be downloaded from their respective websites.  Ansible is a Python package with [many ways to install](http://docs.ansible.com/intro_installation.html).  One method is to use [Homebrew](http://brew.sh) (which, in turn, requires ruby) and simply `brew install ansible`.
 
 These packages are available on many platforms, but the Dryad organization primarily uses them on recent versions of Mac OS X.
 
 ## Getting started
 
-First, You will need to clone the repository:
+You will need to clone the repository that contains the Vagrant/Ansible settings:
 
     git clone git@github.com:datadryad/vagrant-dryad.git
     cd vagrant-dryad
 
-Second, you will need to copy the `ansible-dryad/group_vars/all.template` to `ansible-dryad/group_vars/all` and set a database password and repository location.
+## Creating a database
 
-    cp ansible-dryad/group_vars/all.template ansible-dryad/group_vars/all
-    edit ansible-dryad/group_vars/all
+An external machine with a database is required for installation on AWS. On postgres, a local database will be created, but you are welcome to use an external database instead. 
+
+To create a database using Amazon RDS, login to an existing Dryad machine, and run the utility script to create a new database:
+
+    dryad-utils/aws-tools/db-create-new.sh <db-hostname> <db-password>
```

Comment:
I know we're going to do more work to clean this up and I'll put in a better one, but for consistency, this does not yet exist and it might be best to just instruct people to use the local database for now.

Reply:
It's in an open PR

Reply:
https://github.com/datadryad/dryad-utils/pull/35